v2/slogger/rolling_file_appender/rolling_file_appender.go

```diff
@@ -357,15 +399,15 @@ func (self *RollingFileAppender) removeMaxRotatedLogs() error {
 
 const MAX_ROTATE_SERIAL_NUM = 1000000000
 
-func (self *RollingFileAppender) renameLogFile(oldFilename string) error {
+func (self *RollingFileAppender) renameLogFile(oldFilename string) (string, error) {
```

Comment:
Please document (preferably with a variable name as opposed to a comment as comments tend to go stale) what this new return string is.

Reply:
ack. Due to change in compression though allowing for a specified number of uncompressed files, changes to this method signature have been reverted